src/components/VersionChooser/index.tsx

```diff
@@ -36,6 +40,22 @@ type Props = PublicProps &
   PropsFromState &
   RouterProps;
 
+export const higherVersionsThan = (versionId: string) => {
+  if (!versionId) {
```

Comment:
How about `if (versionId === '')` ? I think it could also benefit from a comment saying that the source is a query parameter.

Reply:
I will try. I am not sure it will fix the storybook issue though, which is the main point of having this check. Otherwise, we should never have an empty string, everything else would break before :D

Reply:
Na, it does not work as intended. I see why you asked for `''` now. The test case is another issue: TS does not want to let me call this function with `undefined` or `null`, only an empty string works here.

As said in my 3 previous comments above, this is mainly for Storybook, which does not pass any routing params, and I don't really know how to simulate that (other than using the `Route` component). Passing `match` to the component does not work.